internal/dataplane/kongstate/route.go

```diff
@@ -24,7 +24,8 @@ type Route struct {
 }
 
 var (
-	validMethods = regexp.MustCompile(`\A[A-Z]+$`)
+	validMethods      = regexp.MustCompile(`\A[A-Z]+$`)
```

Comment:
I considered it, but it'd become both something we have to remember to update and something that breaks Kong/KIC version compatibility in a rather silly way: if we add v2 in the future, I wouldn't want all older KIC versions to be locked out just because that value is unrecognized.